benchmark/studies/batch_size.rb

```diff
@@ -0,0 +1,33 @@
+require "fileutils"
+
+require_relative "../ghostferry_benchmark.rb"
+
+row_sizes = [1, 100, 200, 500]
+batch_sizes = (5000..10000).step(1000).to_a
+
+max_total_bytes = 100_000_000
```

Comment:
Is this the max packet size that can be written to mysql ?

Reply:
No. It's just a maximum byte size I want to insert to the source so the benchmark doesn't run forever.